### PR TITLE
Use 'epel-7-x86_64' mock target to have Golang on CentOS

### PR DESCRIPTION
**Pull request by lebauce**
skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-k8s-tests
skip skydive-vpp-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
rerun skydive-vagrant-tests

File: scripts/ci/run-packaging-tests.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ VERSION=$(make -s version | cut -d '-' -f 1)
 TAG=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
 for srpm in $(ls rpmbuild/SRPMS/skydive-${VERSION}*${TAG}.src.rpm)
 do
-    for root in centos-7-x86_64 epel-7-x86_64 fedora-27-x86_64 fedora-28-x86_64
```
Comment (nplanel) [inline scripts/ci/run-packaging-tests.sh]: If I remember well centos mock was used to test some selinux package dependency, I think epel have container-selinux pkg but centos/rhel don't or something like that

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
run skydive-vagrant-tests

**Comment by safchain** [comment]
rerun skydive-vagrant-tests

**Comment by lebauce** [comment]
run skydive-vagrant-tests

**Comment by nplanel** [comment]
run skydive-vagrant-tests

**Pull request merged**